presto-cli/src/main/java/com/facebook/presto/cli/LdapRequestFilter.java

```diff
@@ -1,48 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.facebook.presto.cli;
-
-import com.google.common.net.HttpHeaders;
-import io.airlift.http.client.HttpRequestFilter;
-import io.airlift.http.client.Request;
-
-import java.util.Base64;
-
-import static com.google.common.base.Preconditions.checkArgument;
-import static io.airlift.http.client.Request.Builder.fromRequest;
-import static java.nio.charset.StandardCharsets.ISO_8859_1;
-import static java.util.Objects.requireNonNull;
-
-public class LdapRequestFilter
```

Comment:
Then i should copy this class to 3 modules. `presto-product-tests`, `presto-jdbc`, `presto-main`. Furthermore, i still need airlift patched in order to register this filter to all of the internal http clients in `presto-main.`.